play-json/shared/src/main/scala-2.13-/play/api/libs/json/ImmutableLinkedHashMap.scala

```diff
@@ -20,8 +20,9 @@ import scala.collection.mutable.ArrayBuffer
 private[json] class ImmutableLinkedHashMap[A, +B](underlying: JLinkedHashMap[A, B])
     extends AbstractMap[A, B]
     with Map[A, B]
-    with MapLike[A, B, ImmutableLinkedHashMap[A, B]] {
+    with MapLike[A, B, ImmutableLinkedHashMap[A, B]] with Serializable {
 
+  private val serialVersionUID        = -2338626292552177485L
```

Comment:
again, by looking at std lib collections you can see the use of `@SerialVersionUID` annotation instead of the field...

Reply:
adjusted